main_project/our_module.rb

```diff
@@ -306,10 +306,7 @@ def create_project_with_name
     create_project_commit
   end
 
-  def open_project(project_name)
-    @wait.until {@driver.find_element(:css => '#projects-index').displayed?}
-    projects = @driver.find_elements(:css => '.project.root.leaf')
-
+  def is_project_in_list?(project_name, projects)
```

Comment:
"is_project_in_list?" method looks for project name (project that has to be opened) in the all existing projects names list.
If there is no such project in the list - it sets @exist to false